src/stream.rs

```diff
@@ -367,47 +391,73 @@ pub fn compress_buf(params: &CompressParams,
 pub fn compress_vec(params: &CompressParams,
```

Comment:
:D  That's the only compression function we use!  It's fine to remove this though - I'll update our code to just use the `write::BrotliEncoder`.